portal/src/main/webapp/sitemap_index.jsp

```diff
@@ -0,0 +1,33 @@
+<%@ page trimDirectiveWhitespaces="true" %>
+<%@ taglib prefix = "c" uri = "http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/xml; charset=UTF-8" %>
+<%@ page import="org.json.simple.JSONArray" %>
+<%@ page import="org.json.simple.JSONObject" %>
+<%@ page import="org.json.simple.parser.*" %>
+<%@ page import="java.util.*" %>
+<%@ page import="org.mskcc.cbio.portal.util.GlobalProperties" %>
+
+<%
+String protocol = (request.isSecure()) ? "https" : "http";
+pageContext.setAttribute("serverRoot", protocol + "://" + "www.cbioportal.org/beta");
+%>
+
+<c:import var="dataJson" url="${serverRoot}/api/studies"/>
+
+<%
+String json = (String)pageContext.getAttribute("dataJson");
+Object obj = new JSONParser().parse(json);
+JSONArray ja = (JSONArray) obj;   
+pageContext.setAttribute("mylist", ja);
+%>
+<c:if test = "${GlobalProperties.showSitemaps()}">
```

Comment:
@inodb for the life of me i can't get showSitemaps to reflect the property (either in portal.properties or from command arg).  Can you help?

Reply:
@alisman 🆗